testData/mock/stub/builtins.py

```diff
@@ -0,0 +1,22 @@
+"""
+The Mock is for testing pydantic with builtins
```

Comment:
👀

Reply:
I have changed the name to `strawberry`.
We will need the unittest. I keep this file.